src/components/studentProgress/CustomAppBar.css

```diff
@@ -0,0 +1,16 @@
+#Custom-AppBar{
```

Comment:
Follow naming conventions like this **#custom-app-bar** everywhere. Don't use it like this **.back-button-InCustomAppBar**